frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedRows.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { t } from "ttag";
+
+import { ChartSettingOrderedItems } from "./ChartSettingOrderedItems";
+
+import {
+  ChartSettingMessage,
+  ChartSettingOrderedRowsRoot,
+} from "./ChartSettingOrderedRows.styled";
+
+interface Row {
+  rowIndex: number;
+  display_name: string;
+  enabled: boolean;
+}
+
+interface ChartSettingOrderedRowsProps {
+  onChange: (rows: Row[]) => void;
+  rows: Row[];
+  value: Row[];
+}
+
+export const ChartSettingOrderedRows = ({
+  onChange,
+  rows,
+  value,
```

Comment:
What do you think about giving this var an alias like `funnelRows` maybe? if we update `fr` to `funnelRow` in `frontend/src/metabase/visualizations/components/FunnelNormal.jsx` then maybe it's more obvious that those are related.

I'm not sure if I understand this correctly, but at least I think we should give it a name otherwise, using `value` inside the function will be somewhat vague since we have no clear picture of what `value` could be.

Here are some usages of `value` below which looks vague
- `[...value]`
- `value.length > 0`

Since, I definitely won't think `value` would be an array.

Reply:
Totally fair. Since this component really doesn't care about the funnel side of things, I'm going to alias value to "orderedRows", so it's clear what it represents